definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-bailiff-nonprod.json

```diff
@@ -439,5 +439,40 @@
     "CaseFieldID": "LabelContinueBecauseOfSuccessfulBailiff",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "08/04/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationType",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRUD"
```

Comment:
I don't know how ExUI or CCD UI would behave if the field is not populated / doesn't exist in cases for search filter so I'd rather do it as loose as possible.

What is more beta or la will trigger an event that populates this field ("make service decision" or "make bailiff decision").